extensions/guacamole-auth-ldap/pom.xml

```diff
@@ -51,12 +51,12 @@
         <dependency>
             <groupId>org.apache.directory.api</groupId>
             <artifactId>api-all</artifactId>
-            <version>2.1.6</version>
+            <version>2.1.7</version>
             <exclusions>
 
                 <!--
                     Replace slightly older commons-lang3 (3.13.0) with latest
-                    compatible version (3.14.0) so that we don't need two copies
+                    compatible version (3.16.0) so that we don't need two copies
                     of the same license information.
                 -->
```

Comment:
Is it still correct (this newer version of `api-all` is still pointing at version 3.13.3 of `commons-lang3`)?

Reply:
Looks like it depends on `3.14.0` and `3.15.0` still 

https://mvnrepository.com/artifact/org.apache.directory.api/api-all/2.1.7

```
Dependency convergence error for org.apache.commons:commons-lang3:3.14.0 paths to dependency are:
+-org.apache.guacamole:guacamole-auth-ldap:1.5.5
  +-org.apache.directory.api:api-all:2.1.7
    +-org.apache.commons:commons-text:1.12.0
      +-org.apache.commons:commons-lang3:3.14.0
and
+-org.apache.guacamole:guacamole-auth-ldap:1.5.5
  +-org.apache.directory.api:api-all:2.1.7
    +-org.apache.commons:commons-lang3:3.15.0
and
+-org.apache.guacamole:guacamole-auth-ldap:1.5.5
  +-org.apache.commons:commons-lang3:3.16.0
```